packages/postgres-database/src/simcore_postgres_database/models/comp_runs.py

```diff
@@ -98,6 +98,9 @@
         nullable=False,
         doc="the run uses on demand clusters",
     ),
+    sa.Column(
+        "dag_adjacency_list", sa.JSON, doc="Adjancey list for the pipeline's graph"
```

Comment:
can you use JSONB instead of legacy JSON? I think the JSONB has almost only advantages.